docs/frontendconfig/index.md

```diff
@@ -0,0 +1,64 @@
+# Frontend configuration
+
+In only two json files SciCat's frontend configuration is managed, both as part of the backend configuration:
+
+1. [frontend.config.json](https://github.com/SciCatProject/scicat-backend-next/blob/master/src/config/frontend.config.json) and 
+2. [frontend.theme.json](https://github.com/SciCatProject/scicat-backend-next/blob/master/src/config/frontend.theme.json)
+
+Please write about
+ - Why the frontend config is in the backend config?
+ - What the scope is of these two files? 
+ - Please provide code snippets examples for enabling/disabling buttons. Add ideally screenshots.
+
+# Default List & Filter Configuration Pattern - Frontend Configuration Guide
+
+## Overview
+
+This guide explains how to configure the List & Side-Panel Configuration Pattern used on the frontend.
+It allows customizing how list-based components (e.g., datasets, proposals) display table columns, side-panel filters, and optional query conditions.
+
+The configuration should be defined or mounted at the location specified by the environment variable `FRONTEND_CONFIG_FILE` (default: `src/config/frontend.config.json`).
```

Comment:
is this for both documents - `frontend.theme.json` and `frontend.config.json`?
Maybe we could be more explicit with an example from docker:
```
env:
   - FRONTEND_CONFIG_FILE: src/config/frontend.config.json
....
volumes:
  - frontend.config.json:/home/src/config/frontend.config.json
  - frontened.theme.json:/home/src/config/frontend.theme.json

Reply:
Also it seems we need a FRONTEND_THEME_CONFIG file as well. Perhaps we could include this in the docs?